api/dps/index_test.go

```diff
@@ -131,413 +123,387 @@ func TestIndex_Last(t *testing.T) {
 }
 
 func TestIndex_Header(t *testing.T) {
-	testHeader := &flow.Header{
-		ChainID: dps.FlowTestnet,
-		Height:  42,
-	}
-
-	testHeaderB, err := cbor.Marshal(testHeader)
+	headerB, err := cbor.Marshal(mocks.GenericHeader)
```

Comment:
Is this supposed to mean `headerBytes`? Would prefer explicit naming here, I was confused a second, because I thought it was something like `headerA` and `headerB`.

Reply:
Oh right, yeah I meant bytes. Will change that then 👍